operatorcert/entrypoints/marketplace_replication.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import Any
+
+from twirp.context import Context
+from twirp.exceptions import TwirpServerException
+from google.protobuf.json_format import Parse
+
+from operatorcert import get_csv_content
+from operatorcert.webhook.webhook import webhook_twirp
+from operatorcert.webhook.webhook import webhook_pb2
+from operatorcert.logger import setup_logger
+
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> Any:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description="Call the IBM webhook to trigger marketplace replication"
+    )
+    parser.add_argument(
+        "--bundle-path",
+        required=True,
+        help="Location of the operator bundle",
+    )
+    parser.add_argument("--git-repo-url", required=True, help="URL of the git repo")
+    parser.add_argument("--package", required=True, help="Operator package name")
+    parser.add_argument(
+        "--ocp-version", required=True, help="OCP versions in the bundle annotations"
+    )
+    parser.add_argument(
+        "--version", required=True, help="Version of the operator bundle"
+    )
+    parser.add_argument(
+        "--webhook-url",
+        default="https://mirroring-mirroring.cicd-us-east-2-bx2-4x16-e45420c0be249b88cfacab5f393b43c1-0000.us-east.containers.appdomain.cloud",
+        help="URL to call the webhook",
+    )
+    parser.add_argument("--verbose", action="store_true", help="Verbose output")
+    return parser
+
+
+def call_ibm_webhook(args: Any) -> None:
+    # Marketplace repos:
+    # https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators-preprod/
+    # https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators/
+    marketplace_repo_regex = re.compile(
+        ".*github\.com\/redhat-openshift-ecosystem\/redhat-marketplace-operators*"
+    )
+    if not marketplace_repo_regex.match(args.git_repo_url):
```

Comment:
Each of our submission repos has the `organization` defined in its `config.yaml` ([example](https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators-preprod/blob/stage/config.yaml#L4)). Can we parse that instead? We can then drop the `git-repo-url` arg.